app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -260,7 +260,7 @@ case class Choice(
   selections: List[Int],
   hints: Option[NonEmptyList[SmartString]],
   optionHelpText: Option[NonEmptyList[SmartString]],
-  dividerPosition: Option[Int],
+  dividerPosition: Option[String],
```

Comment:
Please introduce proper type (ADT) for this.

Reply:
I thought ADT is a bit overkill for this case.